std/assembly/util/math.ts

```diff
@@ -0,0 +1,1919 @@
+/** Lookup data for exp2f **/
```

Comment:
Looks like this would become the doc block of EXP2F_TABLE_BITS when using a documentation generator. Wdyt of using `//` comments for everything that is not intended to become documentation?

Reply:
Yesh, it seems I miss this. Usually I convert /**/ to //